phoenix-core-server/src/main/java/org/apache/phoenix/coprocessor/CDCCompactionUtil.java

```diff
@@ -130,7 +129,6 @@ private static Map<String, Object> createTTLDeleteCDCEvent(Put expiredRowPut, PT
             }
         }
         cdcEvent.put(QueryConstants.CDC_PRE_IMAGE, preImage);
-        cdcEvent.put(QueryConstants.CDC_POST_IMAGE, Collections.emptyMap());
```

Comment:
We don't keep post-image if the event is DELETE, therefore same needs to apply to TTL_DELETE also